pkgs/by-name/pi/piped-frontend/package.nix

```diff
@@ -0,0 +1,57 @@
+{
+  lib,
+  stdenv,
+  fetchFromGitHub,
+  nodejs,
+  pnpm,
```

Comment:
Most likely you need to change `pnpm` to `pnpm_9`. Since #371832, it points to `pnpm_10` instead of `pnpm_9`.